react/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "wbooks",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-scripts": "1.0.17",
+    "roboto-npm-webfont": "^1.0.1",
+    "typeface-roboto": "0.0.45"
```

Comment:
Usually we use https://fonts.google.com/ for font config, but I think this is ok to the scope of the training.